crm-plugin/controllers/tag.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+/**
+ * crm-plugin.js controller
+ *
+ * @description: A set of functions called "actions" of the `crm-plugin` plugin.
+ */
+const { sanitizeEntity } = require("strapi-utils");
+module.exports = {
+  /**
+   * Default action.
+   *
+   * @return {Object}
+   */
+
+  index: async (ctx) => {
+    // Add your own logic here.
+
+    // Send 200 `ok`
+    ctx.send({
+      message: "ok",
+    });
+  },
+
+  find: async (ctx) => {
+    let tag = await strapi.query("tag", "crm-plugin").find(ctx.query);
+    return tag.map((entity) =>
```

Comment:
case for ctx.query._q not handle. please check properly and refer contact.js file for same